recipes/pydeps/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pydeps" %}
+{% set version = "1.10.12" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydeps-{{ version }}.tar.gz
+  sha256: 712a44a211f47b9b5a1ff6e98e2765ed3fbaf926c194ea6081d4893786c576ec
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pydeps = pydeps.pydeps:pydeps
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest
```

Comment:
Is pytest actually needed? I don't see it actually used in `setup.py`, unless `setuptools.command.test` fails without it?

Reply:
Let me try taking it out and see what happens